campusromero_openedx_extensions/custom_registration_form/migrations/0008_customformfields_province.py

```diff
@@ -15,6 +15,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='customformfields',
             name='province',
-            field=models.CharField(blank=True, choices=[(b'Lima', b'Lima'), (b'Amazonas', b'Amazonas'), (b'Ancash', b'Ancash'), (b'Apurimac', b'Apurimac'), (b'Arequipa', b'Arequipa'), (b'Ayacucho', b'Ayacucho'), (b'Cajamarca', b'Cajamarca'), (b'Callao', b'Callao'), (b'Cusco', b'Cusco'), (b'Huancavelica', b'Huancavelica'), (b'Huanuco', b'Huanuco'), (b'Ica', b'Ica'), (b'Junin', b'Junin'), (b'La-libertad', b'La Libertad'), (b'Lambayeque', b'Lambayeque'), (b'Loreto', b'Loreto'), (b'Madre-de-Dios', b'Madre de Dios'), (b'Moquegua', b'Moquegua'), (b'Pasco', b'Pasco'), (b'Piura', b'Piura'), (b'Puno', b'Puno'), (b'San-martin', b'San Martin'), (b'Tacna', b'Tacna'), (b'Tumbes', b'Tumbes'), (b'Ucayali', b'Ucayali')], max_length=20, null=True, verbose_name='Province'),
+            field=models.CharField(choices=[(b'Lima', b'Lima'), (b'Amazonas', b'Amazonas'), (b'Ancash', b'Ancash'), (b'Apurimac', b'Apurimac'), (b'Arequipa', b'Arequipa'), (b'Ayacucho', b'Ayacucho'), (b'Cajamarca', b'Cajamarca'), (b'Callao', b'Callao'), (b'Cusco', b'Cusco'), (b'Huancavelica', b'Huancavelica'), (b'Huanuco', b'Huanuco'), (b'Ica', b'Ica'), (b'Junin', b'Junin'), (b'La-libertad', b'La Libertad'), (b'Lambayeque', b'Lambayeque'), (b'Loreto', b'Loreto'), (b'Madre-de-Dios', b'Madre de Dios'), (b'Moquegua', b'Moquegua'), (b'Pasco', b'Pasco'), (b'Piura', b'Piura'), (b'Puno', b'Puno'), (b'San-martin', b'San Martin'), (b'Tacna', b'Tacna'), (b'Tumbes', b'Tumbes'), (b'Ucayali', b'Ucayali')], max_length=20, null=True, verbose_name='Province'),
```

Comment:
Why remove the `blank=True`? And why the byte strings?

Reply:
I eliminated the `blank=true` so that in the registration form the province field would appear, with the `blank=true` it ignores the province field.
And the string byte, this is the migration file and I didn't touch it.